### PR TITLE
feat: added support for Gemini 1.5 Flash

### LINKED ISSUE #87: Add support Gemini 1.5 Flash

### PR DESCRIPTION
**Pull request by roman-romanov-o**
Closes https://github.com/epam/ai-dial-adapter-vertexai/issues/87

### CODE REVIEW

File: aidial_adapter_vertexai/deployments.py [hunk 1]
```diff
@@ -14,6 +14,7 @@ class ChatCompletionDeployment(str, Enum):
     GEMINI_PRO_1 = "gemini-pro"
     GEMINI_PRO_VISION_1 = "gemini-pro-vision"
     GEMINI_PRO_1_5 = "gemini-1.5-pro-preview-0409"
+    GEMINI_FLASH_1_5 = "gemini-1.5-flash"
```
Comment (roman-romanov-o) [inline aidial_adapter_vertexai/deployments.py]: Not sure, what version to add here for prod

In docs there is three options:

- Latest version: `gemini-1.5-flash-latest`
- Latest stable version: `gemini-1.5-flash`
- Stable version: `gemini-1.5-flash-001`

**Comment by roman-romanov-o** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/483790)

**Comment by roman-romanov-o** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/483836)

File: aidial_adapter_vertexai/deployments.py (hunk 1 above)
Comment (adubovik) [reply aidial_adapter_vertexai/deployments.py]: Let's use the stable version `gemini-1.5-flash-001`.

**Comment by adubovik** [comment]
Please update README

File: aidial_adapter_vertexai/chat/gemini/adapter.py [hunk 2]
```diff
@@ -111,6 +114,10 @@ async def parse_prompt(
                 return await Gemini_1_5_Pro_Prompt.parse(
                     self.file_storage, tools, messages
                 )
+            case ChatCompletionDeployment.GEMINI_FLASH_1_5:
```
Comment (adubovik) [inline aidial_adapter_vertexai/chat/gemini/adapter.py]: No need to complicate the code by introduction of the alias class `Gemini_1_5_Flash_Prompt`.

```python
            case (
                ChatCompletionDeployment.GEMINI_PRO_1_5
                | ChatCompletionDeployment.GEMINI_1_5_FLASH
            ):
                return await Gemini_1_5_Pro_Prompt.parse(
                    self.file_storage, tools, messages
                )
```

Comment (roman-romanov-o) [reply aidial_adapter_vertexai/chat/gemini/adapter.py]: Ok, maybe then let's rename this prompt parser to something like `Gemini_1_5_Prompt`?

**Comment by roman-romanov-o** [comment]
> Please update README

done

File: aidial_adapter_vertexai/deployments.py (hunk 1 above)
Comment (roman-romanov-o) [reply aidial_adapter_vertexai/deployments.py]: Done

File: aidial_adapter_vertexai/chat/gemini/adapter.py (hunk 2 above)
Comment (roman-romanov-o) [reply aidial_adapter_vertexai/chat/gemini/adapter.py]: Fixed, renamed `Gemini_1_5_Pro_Prompt` to `Gemini_1_5_Prompt`

**Comment by roman-romanov-o** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-adapter-vertexai-pr-88.nightly-test.deltixhub.io
>E2E tests status: success

**Review by adubovik — APPROVED** [review]

**Pull request merged**